app/views/overview_application_processes/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= provide(:title, 'Application Process - Common fields') %>
```

Comment:
Minor point - let's also add SEO tags as we're going through use the meta-tags gem

Reply:
You can pass a hash of values